cobbler/actions/buildiso/__init__.py

```diff
@@ -48,27 +48,37 @@ def add_remaining_kopts(kopts: Dict[str, Union[str, List[str]]]) -> str:
     return " ".join(append_line)
 
 
-class BootFilesCopyset(NamedTuple):
+class BootFilesCopyset(NamedTuple):  # pylint: disable=missing-class-docstring
     src_kernel: str
     src_initrd: str
     new_filename: str
 
 
-class LoaderCfgsParts(NamedTuple):
+class LoaderCfgsParts(NamedTuple):  # pylint: disable=missing-class-docstring
     isolinux: List[str]
     grub: List[str]
     bootfiles_copysets: List[BootFilesCopyset]
 
 
-class BuildisoDirs(NamedTuple):
+class BuildisoDirsX86_64(
+    NamedTuple
+):  # noqa: N801 pylint: disable=invalid-name,missing-class-docstring
```

Comment:
[nit] What's N801? Can we disable it using its name?

Reply:
N801 is raised by `pep8-naming` while running flake8 (during Codacy analysis): https://github.com/PyCQA/pep8-naming#error-codes

For flake8, apparentely we cannot use its name but only the specific code: https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors

Reply:
Thanks! I didn't find N801 in the pylint list and didn't quickly find out what other checkers/error sources are used.